Sources/Foundation/FileManager+POSIX.swift

```diff
@@ -471,6 +471,18 @@ extension FileManager {
         return self.string(withFileSystemRepresentation: buf, length: Int(len))
     }
     
+    internal func _recursiveDestinationOfSymbolicLink(atPath path: String) throws -> String {
+        let _resolvedPath = try _fileSystemRepresentation(withPath: path) {
+            realpath($0, nil)
```

Comment:
If `nil` as passed the buffer then it is `malloc()`ed so will need to be `free()`d if the call is successful

Reply:
Fixed.